dashboard/app/models/peer_review.rb

```diff
@@ -133,4 +139,14 @@ def self.create_for_submission(user_level, level_source_id)
       end
     end
   end
+
+  def summarize
+    return Hash.new.tap do |summary_hash|
+      summary_hash[:id] = id
+      summary_hash[:status] = status.nil? ? 'attempted' : 'perfect'
+      summary_hash[:name] = status.nil? ? I18n.t('peer_review.review_in_progress') : I18n.t('peer_review.link_to_submitted_review')
+      summary_hash[:result] = status.nil? ? ActivityConstants::UNSUBMITTED_RESULT : ActivityConstants::BEST_PASS_RESULT
+      summary_hash[:icon] = status.nil? ? '' : 'fa-check'
+    end
+  end
```

Comment:
I was doing this with tap because I didn't want to set the icon field for in progress reviews. But that's no longer the case, so I'll do this method